plugins/modules/zos_copy.py

```diff
@@ -114,7 +114,7 @@
         be deleted and recreated following the process outlined in the C(volume) option.
       - When the C(dest) is an existing VSAM (RRDS), then the source must be an RRDS.
         The VSAM (RRDS) will be deleted and recreated following the process outlined
-        in the C(volume) option.
+       in the C(volume) option.
```

Comment:
Have you tried generating docs after this change? Dunno if a single space would actually break something, but you never know with YAML

Reply:
No change but it was part of the issues of sanity test that space

Reply:
This one is weird, I did run the html generation and looks fine. What was the sanity issue raised due to this space?